components/dataComponents/contactdata.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable jsx-a11y/alt-text */
+import React from 'react'
+
+export default function ContactData({
+  linkType,
+  text,
+  imagePath,
+}: {
+  linkType: string
+  text: string
+  imagePath: string
+}) {
+  let Type: string
```

Comment:
use camelcase for variables. We use pascal case for component names and types and interfaces.
You have mix styles: Type = linkType, it should be type = linkType.

Reply:
fixed, change Type to type